core/state/trie_prefetcher.go

```diff
@@ -257,9 +256,7 @@ func newSubfetcher(db Database, root common.Hash, accountHash common.Hash) *subf
 		seen:        make(map[string]struct{}),
 		accountHash: accountHash,
 	}
-	gopool.Submit(func() {
-		sf.loop()
```

Comment:
why this change? irrelevant with pipe commit?

Reply:
Yes, a bit overhead with `gopool.Submit`